test/blackbox-tests/test-cases/ocaml-config-ox.t/run.t

```diff
@@ -0,0 +1,5 @@
+The macro %{ocaml-config:ox} should be available. we don't print anything because
```

Comment:
Could you turn this into a file cram test? See some of the other file cram tests to get an idea of how to write the files. It makes it easier to see what the test is about in one place.

Reply:
Thanks, that does indeed look clearer!